packages/database/prisma/schema.prisma

```diff
@@ -180,6 +180,7 @@ model Price {
 model Product {
   id               String               @id @default(uuid())
   name             String
+  productType      String
```

Comment:
This is going to have to start as `productType    String?` in order to be able to migrate against a database that already has records in the `Products` table. Without the `?` on `String?`, the column is `not null`. That non-nullable constraint will be violated by the migration and it will be forced to rollback.

A good way of thinking about how to apply this kinds of schema changes is [the Expand and Contract pattern](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern#:~:text=various%20database%20transformations.-,What%20is%20the%20expand%20and%20contract%20pattern%3F,data%20structure%20without%20affecting%20uptime.).

Typically that means the migration will have to be multi-step:
1. Add new nullable field and deploy
2. Update any _write_ app code to apply some param or default value when creating `Product` records
3. Backfill existing records with whatever data should go in that field
4. Update schema to make the field non-nullable and deploy

Another way to approach this would be to keep it non-nullable and give it a default value of `self-paced`.

Did you run in to this non-nullable migration rollback with your development database while implementing this?